git-extra/notepad

```diff
@@ -15,7 +15,12 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config i18n.commitencoding)" ||
+		commitencoding=""
```

Comment:
There is no explicit documentation, that `git config` doesn't output anything to `stdout` or that all errors are written to `stderr`, but [Documentation/technical/api-error-handling.txt](https://github.com/git/git/blob/master/Documentation/technical/api-error-handling.txt) strongly suggests that errors should be written to `stderr`. [`stdio.h`](https://github.com/lattera/glibc/blob/master/libio/stdio.h#L167-L174), [stdio(3)] and [stderr(3)] suggest the same behaviour. I don't have POSIX/SUS/LSB/C89/C90/C95/C99/C11 standards at hand right now, but I'm sure they suggest similar behaviour. And what non-error information should git config output if it doesn't find a key or section?